### PR TITLE
Upgrade jest-image-snapshot

### PR DESCRIPTION
**Pull request by RoyEJohnson**
For: https://github.com/openstax/unified/issues/2512

### CODE REVIEW

File: src/app/content/search/components/SearchResultsSidebar/RelatedKeyTerms.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const RelatedKeyTerms = ({ book, keyTermHits, selectedResult }: {
       book={book}
       hits={keyTermHits}
       testId='related-key-term-result'
-      getPage={(hit) => assertDefined(findArchiveTreeNodeById(book.tree, hit.source.pageId), 'hit has to be in a book')}
+      getPage={(hit: SearchResultHit) => assertDefined(findArchiveTreeNodeById(book.tree, hit.source.pageId), 'hit has to be in a book')}
```
Comment (jivey) [inline src/app/content/search/components/SearchResultsSidebar/RelatedKeyTerms.tsx:29]: That's odd that the type inference stopped working, I wonder why 🤔

Comment (RoyEJohnson) [reply src/app/content/search/components/SearchResultsSidebar/RelatedKeyTerms.tsx:29]: It is weird. I wonder whether having TS fully upgraded would make a difference?

**Comment by jivey** [comment]
@RoyEJohnson This one slipped off my radar, should I take another look? I think you'd mentioned something about the typescript upgrade affecting it (which I think was merged?) Just making sure I'm not holding anything up.

File: src/app/content/search/components/SearchResultsSidebar/RelatedKeyTerms.tsx (hunk 1 above)
Comment (RoyEJohnson) [reply src/app/content/search/components/SearchResultsSidebar/RelatedKeyTerms.tsx:29]: @jivey The upgrade-typescript PR is approved but not merged. I tried upgrading TS locally and taking out the type declaration, and it is still a problem. It is likely related to one of the other upgrades in this PR. The update of @types/react-redux was committed with three type declarations; it is likely the reason the declarations became an issue.

**Review by jivey — APPROVED** [review]

**Pull request merged**